partials/crmMailingAB/report.html

```diff
@@ -1,13 +1,17 @@
 <!--
   Implicit Controller: CrmMailingABEditCtrl
 -->
+<div class="messages help">
+  <div class="msg-title crm-title">{{ts('A/B Test Results')}}: {{abtest.ab.name}}</div>
+  {{ts('This report displays the current results for your A/B test. You can return to this page to view the latest statistics by clicking "Results" from the "Manage A/B Tests" listing.')}}
```

Comment:
I'd put the instructions in the order in which you do them; change from:

```
by clicking "Results" from the "Manage A/B Tests" listing.
```

to

```
by navigating to "Manage A/B Tests" and clicking "Results".
```